rustls/src/webpki/verify.rs

```diff
@@ -275,7 +275,7 @@ impl WebPkiClientVerifier {
     /// Construct a new `WebpkiClientVerifier`.
     ///
     /// `roots` is the list of trust anchors to use for certificate validation.
-    /// `crls` are an iterable of owned certificate revocation lists (CRLs) to use for
+    /// `crls` are a vec of owned certificate revocation lists (CRLs) to use for
```

Comment:
I would capitalize `Vec`. (I would also write `is` rather than `are`, and maybe this deserves a bullet list, too?)

Reply:
Fixed up.